back/config/schemas/settings.schema.json.erb

```diff
@@ -550,18 +550,6 @@
         }
       },
 
-      "participatory_budgeting": {
```

Comment:
The clean_tenant_settings task is run during deployment in CI. It will remove features from existing tenants that are no longer in the tenant settings schema.